plat/linuxu/arm64/entry64.S

```diff
@@ -0,0 +1,45 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Authors: Florin Postolache <florin.postolache80@gmail.com>
+ *
+ *
+ * Copyright (c) 2022, Florin Postolache, <florin.postolache80@gmail.com>
```

Comment:
I should probably add some more comments to Alex's comment about attribution: We usually keep the original attribution for files that were copied from somewhere else and got then modified by us. For the files that you copied from linuxu-x86 (if there are any), you should keep NEC and add yourself additionally. It is fine to have multiple copyright lines.